code/modules/mob/living/simple_animal/parrot.dm

```diff
@@ -908,4 +929,18 @@
 	S["phrases"] 			<< speech_buffer
 	S["roundssurvived"]		<< rounds_survived
 	S["longestsurvival"]	<< longest_survival
+	S["longestdeathstreak"] << longest_deathstreak
 	memory_saved = 1
+
+/mob/living/simple_animal/parrot/Poly/ghost
+	name = "The Ghost of Poly"
+	desc = "Doomed to squawk the earth."
+	color = "#FFFFFF77"
+	speak_chance = 20
+	status_flags = GODMODE

```

Comment:
It might not catch gibbing/dusting or the wand of death but outside of that it's actually surprisingly stable.

By design the ghost of poly will usually be with the one person on the station who could easily dispose of the ghost of poly even with GODMODE on.